app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -119,6 +119,46 @@ def self.default_blacklisted_event_names
     Settings.ems.ems_amazon.blacklisted_event_names
   end
 
+  def self.params_for_create
+    {
+      :title  => "Configure AWS",
+      :fields => [
+        {
+          :component  => "text-field",
+          :name       => "region",
+          :label      => "Region",
+          :isRequired => true,
+          :validate   => [{:type => "required-validator"}]
+        },
+        {
+          :component  => "text-field",
+          :name       => "endpoints.default.access_key",
+          :label      => "Access Key",
+          :isRequired => true,
+          :validate   => [{:type => "required-validator"}]
+        },
+        {
+          :component  => "text-field",
+          :name       => "endpoints.default.secret_access_key",
+          :label      => "Secret Key",
+          :type       => "password",
+          :isRequired => true,
+          :validate   => [{:type => "required-validator"}]
+        },
+        {
+          :component => "text-field",
+          :name      => "endpoints.default.proxy_uri",
+          :label     => "Proxy URI"
+        },
+        {
+          :component => "text-field",
+          :name      => "endpoints.default.assume_role",
+          :label     => "Assume Role"
+        }
+      ]
+    }
```

Comment:
Minor, but I feel this should be a constant, which after that I don't have a preference if it's exposed as a method as well and/or just the constant.